### PR TITLE
transformations: add contrast

### PR DESCRIPTION
**Pull request by manorit2001**

### CODE REVIEW

**Comment by parth-07** [comment]
Please rebase this PR on top of master.

**Comment by manorit2001** [comment]
> Please rebase this PR on top of master.

Done

File: lib/Transformations.cpp [hunk 1]
```diff
@@ -19,6 +19,24 @@ namespace CIL {
         }
     }
 
+    void changeContrast(ImageInfo& img, const double contrast)
+    {
+        double temp = 259 * (contrast + 255) / (255 * (259 - contrast));
+        for (auto px : img)
+        {
+            for (auto i = 0; i < px.numComponents(); i++)
```
Comment (parth-07) [inline lib/Transformations.cpp:27]: Should we add any extra conditions here for the alpha channel?

Comment (manorit2001) [reply lib/Transformations.cpp]: I am not changing the alpha channel anymore

**Review by parth-07 — APPROVED** [review]
LGTM.

**Pull request merged**